applications/cheersland.md

```diff
@@ -214,6 +223,16 @@ Source codes will reside in: https://github.com/cheersland
 * **FTE:**  4
 * **Costs:** $4,000 DAI
 
+**Goal - Implement Multi-asset Staking Platform and the second game.**  
+We will develop the multi-asset staking module and deploy the Staking Contract on Plasm. When users stake tokens as required, the contract will count HPs based on their staking status. All contracts will be released and used after the audit is completed. We will develop the Multi-asset Staking Platform and open several staking pools including DOT and KSM first. Players can earn HPs by staking a certain amount of DOT or KSM and use them in the game.  
+Our game dev team will develop a new classic game for the Polkadot ecosystem, similar to the Crypto Pirates deployed on BSC, which will adopt the same business model and anti-cheating algorithm.  
+
+We will optimize UI and allow players to have a more cheerful gaming experience.  
+**Therefore, under this milestone, our deliverables include:**  
+- Multi-asset Staking Contract 
+- Multi-asset Staking Platform supporting staked asset such as DOT and kSM 
+- Classic games deployed in the Polkadot ecosystem

```

Comment:
Which games? Please be specific

Reply:
In order for the platform to quickly enter the efficient operation phase, we choose classic mini games as the prioritized type of games for launching.  We have launched the first Classic Mini Game, Crypto Pirates, deployed on BSC while it's in early access. We will also deploy Crypto Pirates in the Polkadot ecosystem. 
Know about the gameplay of [Crypto Pirates](https://medium.com/cheersland/how-to-play-crypto-pirates-in-cheersland-789cd3e2d73e). 

We will launch Click Click, Gold Miner, Block Builder and other classic games in next quarters. 
You can read [here ](https://docs.cheersland.org/features/gameplay/multi-game-universe) to know about the games. 

Classic mini games have a broad user base at any age around the world, and allow players to play the game at any time and anywhere with no barriers to participation. This is the reason why we chose to launch classic mini games.

Reply:
ok, please include this list in your deliverables

Reply:
No problem sir, we have submitted a new commit as below.
https://github.com/w3f/Grants-Program/pull/618/commits/4a5e8b539f414133ad15afdcb6aec0ff36969762